reflex-dom-core/src/Reflex/Dom/Prerender.hs

```diff
@@ -235,7 +235,7 @@ instance (SupportsStaticDomBuilder t m) => Prerender JS' t (StaticDomBuilderT t
     _ <- commentNode $ CommentNodeConfig startMarker Nothing
     a <- server
     _ <- commentNode $ CommentNodeConfig endMarker Nothing
-    pure $ pure a
```

Comment:
This change seems unrelated to the fix.

Reply:
Perhaps you could break it out into two commits?

Reply:
Good point. @cgibbard ^

Reply:
If I knew how to use git well enough, perhaps.

Reply:
I can do it for you.

Reply:
Well now my name is on the commit. Rebase to get you name back on the commits.